man/syncthing-config.5

```diff
@@ -98,7 +98,7 @@ The following shows an example of the default configuration file (IDs will diffe
         <disableTempIndexes>false</disableTempIndexes>
         <paused>false</paused>
         <weakHashThresholdPct>25</weakHashThresholdPct>
-        <markerName>.stfolder</markerName>
+        <markerName>.syncthing</markerName>
```

Comment:
Man pages are auto-generated from docs, i.e. these man page changes should happen in https://github.com/syncthing/docs instead.